lib/note-content-editor.tsx

```diff
@@ -152,6 +152,7 @@ class NoteContentEditor extends Component<Props> {
 
   editorReady: EditorDidMount = (editor, monaco) => {
     this.editor = editor;
+    window.editor = editor;
```

Comment:
this was here before for debugging. mind adding a `@TODO` comment to make it easy to find before final merge? (I don't think we want to leave it hanging on the `window` object even though it may not be the worst thing)